### ISSUE TITLE
Accessibility: Calendar component not supporting aria-label and title attributes

### ISSUE THREAD

**Issue by manidevzen3** [primefaces/primeng#6805]
**I'm submitting a ...**  (check one with "x")
```
[ x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
In the context of Accessibility any focused interactive element should have aria-label or title. Calendar component is having text input as focused element which doesn't have support to set aria-label or title

**Expected behavior**
should able to set the aria-label or title to  input element of Calendar component

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**
to make the calendar component accessible to disabled people also

**Please tell us about your environment:**

* **Angular version:** 4.1.3

* **PrimeNG version:** 4.2.1
